consensus/metrics.go

```diff
@@ -81,12 +88,30 @@ func PrometheusMetrics(namespace string, labelsAndValues ...string) *Metrics {
 			Name:      "validators",
 			Help:      "Number of validators.",
 		}, labels).With(labelsAndValues...),
+		LastSignedHeight: prometheus.NewGaugeFrom(stdprometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: MetricsSubsystem,
+			Name:      "last_signed_height",
```

Comment:
I think `validator_` prefix is necessary as consensus scope does not necessary mean that. See https://docs.tendermint.com/master/tendermint-core/metrics.html